lib/src/seek_rewind_button.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+class SeekRewindButton extends StatefulWidget {
```

Comment:
This should be renamed to `SeekControlButton`, so that it better reflects its stated purpose.

That way, you can create buttons that just configure that widget for its specific purpose:

```dart
class SeekRewindButton extends StatefulWidget {
   const SeekRewindButton({
     super.key,
    required this.backgroundColor,
    this.iconColor,
    required this.show,
    this.onPressed,
    this.onDoublePressed,
    this.icon = const Icons.fast_rewind,
   });
   
  final Color backgroundColor;
  final Color? iconColor;
  final bool show;
  final VoidCallback? onPressed;
  final VoidCallback? onDoublePressed;
  final IconData icon;
   
   Widget build(BuildContext context) {
     return SeekControlButton(
       backgroundColor: backgroundColor,
       iconColor: iconColor,
       show: show,
       icon: icon,
       onPressed: onPressed,
       onDoublePressed: onDoublePressed,
     );
   }
}
```

Do likewise for the fast-forward button: (i.e. `SeekFastForwardButton`).